### ISSUE TITLE
Issues in print command with different OS inputs.

### ISSUE THREAD

**Issue by Tideo** [vpython/glowscript#155]
Issue: print('Hello world') does not produce an output when the command is inputted by a iPad Pro 2021.

![image](https://user-images.githubusercontent.com/7073071/132143930-a75877b1-9db6-4e0c-ada1-fb4724dfa4ff.png)

![image](https://user-images.githubusercontent.com/7073071/132143918-01a18a46-d909-4da1-a5d6-b610e7466a86.png)

Line 3 will not produce an output but line 5 will. I have not confirmed if this is the case with _all_ mac devices however I have replicated the issue in multiple ways.

For instance first attempting to assign a string of text to a variable and then printing that variable.

Eg. a = 'hello world'
print(a)

When I type string with an iPad, the string of text does not turn "green" . Whereas if I type strings in other devices such as android, pc, etc. It would turn green. Such as the green text in print('Hello').

![image](https://user-images.githubusercontent.com/7073071/132143912-1587e35c-e7a0-47d8-83bf-445c47c2285a.png)

My university does use the GlowScript IDE and VPython and my peers and I have confirmed that this is an issue. Any help regarding this would be appreciated.

**Comment by BruceSherwood** [comment]
At the moment I don't have access to an iPad (I will in a couple of days). How do you create the curly quotes on an iPad? Is there some reason you can't use straight quotes, either single or double? Also, I'm not clear what happens when you create a = 'hello world' and then try to print(a); does that work or not?

**Comment by BruceSherwood** [comment]
Actually, the answer to your question has only one possible answer, which is to use straight single or double or triple quotes, not curly quotes, which the Python programming language does not handle, nor to my knowledge does any other programming language handle curly quotes. This is similar to the fact that, for historical reasons, programming languages require that a decimal point be expressed using a period, despite the fact that in Europe the decimal point is expressed (outside of programming languages) with a comma. Colorization is a separate issue and there are unavoidable issues with text editors on different operating systems.

**Comment by Tideo** [comment]
I apologize for my lack of programming experience, once you mentioned the curly quotes I noticed that the device I was using defaults to those key characters and I turned off a thing called "smart punctuation" on ipad and voila! It works well now.

There were no colorization issues I believe, simply the usage of the curly quotes resulted in the issues. Even attempting to use a single quote ended up being a single curly quote(Am I saying that right?).
